stream-applications-build/pom.xml

```diff
@@ -32,26 +32,24 @@
         <checkstyle.nohttp.file>${checkstyle.location}/nohttp-checkstyle.xml</checkstyle.nohttp.file>
         <checkstyle.additional.suppressions.file>${checkstyle.location}/checkstyle-suppressions.xml
         </checkstyle.additional.suppressions.file>
-        <nohttp-checkstyle.version>0.0.10</nohttp-checkstyle.version>
+        <nohttp-checkstyle.version>0.0.11</nohttp-checkstyle.version>
         <disable.nohttp.checks>true</disable.nohttp.checks>
-        <spring-javaformat-checkstyle.version>0.0.35</spring-javaformat-checkstyle.version>
-        <spring-boot.version>3.3.1</spring-boot.version>
-        <spring.version>6.1.10</spring.version>
-        <spring-cloud.version>2023.0.2</spring-cloud.version>
-        <spring-cloud-starters.version>4.1.2</spring-cloud-starters.version>
-        <spring-cloud-function.version>4.1.2</spring-cloud-function.version>
-        <spring-cloud-stream-dependencies.version>4.1.2</spring-cloud-stream-dependencies.version>
+        <spring-javaformat-checkstyle.version>0.0.43</spring-javaformat-checkstyle.version>
+        <spring-boot.version>3.2.12</spring-boot.version>
```

Comment:
We can't downgrade to a minor version of Boot in a patch release.

Reply:
I prepared with the same versions as in spring-functions-catalog so that Artem can see if he can find the error.

Reply:
Might be very likely that the problem comes from Debezium itself: they are very prone to ignore compatibility between even between patch versions. I mean we look into some set of Spring versions where it could be just narrowed to Debezium itself .